.env.dist

```diff
@@ -1,9 +1,12 @@
 USER_ID=
-TOKEN=
+CLIENT_TOKEN=
 URL=
 PORT=
```

Comment:
I believe `PORT` is still unused 😬

Reply:
Aaahh, I see. I removed it in ".env", not in ".env.dist" file